agent/pkg/resolver/resolver.go

```diff
@@ -52,6 +53,9 @@ func (resolver *Resolver) GetMap() cmap.ConcurrentMap {
 }
 
 func (resolver *Resolver) CheckIsServiceIP(address string) bool {
+	if resolver == nil {
```

Comment:
@nimrod-up9 added to prevent the runtime error below:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x201435b]

goroutine 15 [running]:
mizuserver/pkg/resolver.(*Resolver).CheckIsServiceIP(...)
	/home/mertyildiran/Documents/up9/mizu/agent/pkg/resolver/resolver.go:56
mizuserver/pkg/api.CheckIsServiceIP(...)
	/home/mertyildiran/Documents/up9/mizu/agent/pkg/api/main.go:160
main.filterItems(0xc000044360, 0xc000044480, 0xc0000a2cc0)
	/home/mertyildiran/Documents/up9/mizu/agent/main.go:230 +0xdb
created by main.main
	/home/mertyildiran/Documents/up9/mizu/agent/main.go:92 +0x39d
```

I don't know why `k8sResolver` becomes `nil` sometimes but I have moved that check with a more appropriate place with a86acefccdd5bed608a21b5ab94cf2b4d97b358e

Reply:
👍